build/extension_manifest.json

```diff
@@ -30,6 +30,6 @@
 		"css": [ "generated.whowrotethat.css" ],
 		"run_at": "document_end"
 	} ],
-	"web_accessible_resources": [ "js/generated.pageScript.js" ],
-	"permissions": [ "activeTab" ]
+	"web_accessible_resources": [ "js/generated.pageScript.js", "js/generated.welcomeTour.js" ],
+	"permissions": [ "activeTab", "storage" ]
```

Comment:
To get this working as a temporary addon in Firefox I had to add the following to `manifest.json`:

	"browser_specific_settings": {
		"gecko": {
			"id": "whowrotethat@wikimedia"
		}
	},

The value of `id` doesn't matter, it just had to be set.